test/operator-framework-e2e/operator_framework_test.go

```diff
@@ -95,892 +46,98 @@ var _ = BeforeSuite(func() {
 	err = operatorv1alpha1.AddToScheme(scheme)
 	Expect(err).ToNot(HaveOccurred())
 
-	err = rukpakv1alpha1.AddToScheme(scheme)
-	Expect(err).ToNot(HaveOccurred())
-
 	c, err = client.New(cfg, client.Options{Scheme: scheme})
 	Expect(err).ToNot(HaveOccurred())
 
 	ctx = context.Background()
+})
 
-	containerRuntime = os.Getenv("CONTAINER_RUNTIME") // This environment variable is set in the Makefile
-	if containerRuntime == "" {
-		containerRuntime = "docker"
+// Since the CATALOG_IMG environment variable is used to configure the
+// tag of the catalog image that is built see if it is specified and use
+// it if it is. Otherwise use the default that is used in setup.sh
+func getCatalogRef() string {
+	if val, exist := os.LookupEnv("CATALOG_IMG"); exist {
+		return val
 	}
+	return "oc-opdev-e2e.operatorframework.io/catalog:e2e"
```

Comment:
This is loosely coupled to `setup.sh` - it might be better to simply enforce that `$CATALOG_IMG` is given as an input.

Reply:
Done as of https://github.com/operator-framework/operator-controller/pull/503/commits/cc277fcd2703a5aaab68e7802480fc8b6fc7412b